.browserslistrc

```diff
@@ -0,0 +1 @@
+> 0.5%, last 2 versions, Firefox ESR, not dead, node 18
```

Comment:
https://github.com/matrix-org/matrix-js-sdk/blob/develop/package.json#L6 we actually target Node 16+

Reply:
Hm that contradicts what https://github.com/matrix-org/matrix-js-sdk#in-nodejs says.
Wondering whether we can claim node 16 support if we don't ship a ponyfill for `fetch` wdyt?

Reply:
As part of constructing a MatrixClient you can pass a `fetchFn`